src/components/Button/LinkButton.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { ComponentRegister } from "../../ComponentRegister";
+import { BaseButtonProps } from "./BaseButton";
+
+import "lbh-frontend/lbh/components/lbh-button/_button.scss";
+
+/**
+ * The proptypes for {@link LinkButton}
+ */
+export interface LinkButtonProps extends BaseButtonProps {
+  href: string;
+  children: React.ReactNode;
+}
+
+/**
+ * The link button, styled as a button, that can be used for routing
+ * within the host application.
+ */
+export const LinkButton: React.FunctionComponent<LinkButtonProps> = (
+  props: LinkButtonProps
+): JSX.Element => {
+  const { id, className, isStartButton, href, children } = props;
+  return (
+    <ComponentRegister.Link
```

Comment:
This has changed the API of `ComponentRegister.Link`. Let's think about whether we want to do that. Also bear in mind that `Component.Link` might not be a wrapper for `<a>`. Does this still work if it's an arbitrary implementation? Do we need a new `ComponentRegister.LinkButton` for this?